Added: Serve command

Juanperias opened the pull request:
This pull request adds the serve command, although the build command would be missing to make it more useful, it uses axum for serve

Juanperias committed:
Added: Serve command

This pull request adds the serve command, although the build command would be missing to make it more useful, it uses axum for serve

SergioRibera commented on src/commands/serve.rs:
```diff
@@ -0,0 +1,45 @@
+
+pub async fn ping() -> Html<&'static str> {
+    Html("<h1>Sergio ribera is a crack</h1>")
+}
```
I appreciate the reference hahahah, but you could make the server provide the files in the corresponding folders, it has to serve the `dest-dir` parameter since the `dir` is the path of the book to be built

Juanperias committed:
Added: Serve command and html render

This commit adds the serve command that axum uses, plus it renders the html as mdbook would do it

SergioRibera reviewed (approved):
Thanks for your contribution
